### PR TITLE
Added localized strings for inline Padding editor

### PR DESCRIPTION
**Pull request by bansalnitish**
In reference to [PR](https://github.com/mozilla/brackets/pull/916)

### CODE REVIEW

File: locales/en-US/editor.properties [hunk 1]
```diff
@@ -218,6 +218,11 @@ PIXEL_UNIT=pixels
 PERCENTAGE_UNIT=percentage
 EM_UNIT=em font unit
 
+# extensions/default/InlinePaddingEditor
+
+ALL_SIDE=All Sides
+INDIVIDUAL_SIDES=Individual Sides
+SET_PADDING_FOR=Set Padding For
 # extensions/default/InlineColorEditor
```
Comment (humphd) [inline locales/en-US/editor.properties]: Can you please insert a blank line above this comment, so it sets off this section visibly?

**Comment by humphd** [comment]
@gideonthomas can you review this please?

**Comment by bansalnitish** [comment]
@humphd I have made the changes here. Can you please review the Quick Edit UI PR in brackets !

File: locales/en-US/editor.properties [hunk 2]
```diff
@@ -218,6 +218,12 @@ PIXEL_UNIT=pixels
 PERCENTAGE_UNIT=percentage
 EM_UNIT=em font unit
 
+# extensions/default/InlinePaddingEditor
+
+ALL_SIDE=All Sides
```
Comment (gideonthomas) [inline locales/en-US/editor.properties]: can we use more meaningful variable names or maybe add a comment above each of these explaining how they will be used?

**Comment by bansalnitish** [comment]
@gideonthomas done !

File: locales/en-US/editor.properties [hunk 3]
```diff
@@ -218,6 +218,15 @@ PIXEL_UNIT=pixels
 PERCENTAGE_UNIT=percentage
 EM_UNIT=em font unit
 
+# extensions/default/InlinePaddingEditor
+# "All Sides" is an option that can be selected for padding in case when users want to change padding value for all the sides at once
+ALL_SIDE=All Sides
+# "Individual Sides" is an option that can be selected for padding in case when users want to change the padding for each side indiviually
```
Comment (gideonthomas) [inline locales/en-US/editor.properties]: make this a single comment otherwise our l10n platform won't pick it up.

Comment (bansalnitish) [reply locales/en-US/editor.properties]: Sure !

**Comment by bansalnitish** [comment]
@gideonthomas I have corrected the comment !

**Comment by humphd** [comment]
Let's make sure we land this before the Brackets PR.

**Pull request merged**